tests/test-generic.php

```diff
@@ -25,6 +25,6 @@ function test_version_exists() {
 	 */
 	function test_debug_mode() {
 
-		$this->assertFalse( OPTML_DEBUG );
+		$this->assertTrue( OPTML_DEBUG );
```

Comment:
Also this should be as before it should be assert false, the tests were failing from the previous comment where we have explicitly `define( 'OPTML_DEBUG', true );` if we revert to the way it was for production it should work as expected.